### ISSUE TITLE
exception w/ no details provided

### ISSUE THREAD

**Issue by johanatan** [scalikejdbc/scalikejdbc#723]
Hi,

I have the following code:

```
try {
  val stmt = org.postgresql.jdbc4.Jdbc4Statement(...)
  // stmt is valid, non-null here
  val db = scalikejdbc.DB(stmt.getConnection)
  println("We do not get here.")
} catch {
  case t: Throwable => println(t.getMessage)
}
```

And it prints merely `scalikejdbc/DB`.

Is there any way to get more descriptive error message out of it?

**Comment by seratch** [comment]
Can you paste the full stack trace (`t.printStackTrace`) at the time, too?

**Comment by johanatan** [comment]
The top frame is the call to `DB`. Everything below that is my app logic/ proprietary.

**Comment by seratch** [comment]
Why do you want to give a connection from an existing statement object? It's not a straightforward way when you use ScalikeJDBC.

**Comment by johanatan** [comment]
Unfortunately I can't really answer that. It is inherited legacy code and the original author is no longer with us.

Is there a more normative way to do this?

**Comment by seratch** [comment]
A statement object should be created from a connection objext beforehand. Use the connection object.

By the way, this is not an issue of the library itself. Please use google group or gitter chat for questions. Do you mind closing this issue?

**Comment by johanatan** [comment]
The issue is the exceedingly unhelpful exception message. That is an actual issue. Feel free to pretend it isn't if you like. Thanks for your time.

**Comment by seratch** [comment]
I’m afraid that the error message doesn’t come from scalikejdbc. If you think this is an issue on scalikejdbc, we need more information or a small sample project to reproduce your situation.

**Comment by seratch** [comment]
Over a year already passed. Let me close this now.

**Comment by johanatan** [comment]
Where do you think the error comes from if not scalikejdbc? What is a user of the library to think/do? I provided the code. Not sure what more I could do.

**Comment by seratch** [comment]
If the error is really raised from scalikejdbc internal code, scalikejdbc always shows stacktrace for it. With the information you provided, I cannot guess what is happening in your project at all.

Now I tried to reproduce your situation but `Jdbc4Statement` is no longer available in the latest postgresql JDBC driver. Also, I tried with older ones and found that the constructor of the class is not public. Is the code you posted a pseudocode? 

Anyway, I won't use my time for this unless you share me a minimal working example which reproduces your situation. Thank you for your understanding.
